px4_ros2_cpp/include/px4_ros2/control/setpoint_types/experimental/trajectory.hpp

```diff
@@ -25,7 +25,11 @@ class TrajectorySetpointType : public SetpointBase
 
   Configuration getConfiguration() override;
 
-  void update(const Eigen::Vector3f & velocity_setpoint_ned_m_s);
+  void update(
+    const Eigen::Vector3f & position_ned_m,
+    const Eigen::Vector3f & velocity_ned_m_s,
+    const Eigen::Vector3f & acceleration_ned_m_s2, float yaw_ned_rad,
+    float yaw_rate_ned_rad_s);
```

Comment:
Can you follow the same pattern as we have for the goto setpoint?
```suggestion
    const std::optional<Eigen::Vector3f> & acceleration_ned_m_s2={}, std::optional<float> yaw_ned_rad = {},
    std::optional<float> yaw_rate_ned_rad_s = {});
```